### PR TITLE
Events

### PR DESCRIPTION
**Pull request by initseis**
In this branch I:

- Used provided files
- Added Big dependency
- Add a function for buttons' click event

math-magicians

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,105 +1,204 @@
-/* eslint-disable react/prefer-stateless-function */
 import React from 'react';
+import calculate from '../logic/calculate';
 
 class Calculator extends React.Component {
```
Comment (arslanbisharat) [inline src/components/Calculator.js]: - [ ] Please import operate.js here just like calculate.js for the app to work fine.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/components/Calculator.js [hunk 2]
```diff
@@ -1,105 +1,204 @@
...
+          >
             .
           </button>
         </div>
         <div className="equal">
-          <button type="button" className="orange-btn">
+          <button
+            type="button"
+            className="orange-btn"
+            onClick={this.handleButton}
```
Comment (arslanbisharat) [inline src/components/Calculator.js:202]: - [ ] When I divide any number by zero instead of giving me an error app stops working.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]

File: src/components/Calculator.js (hunk 2 above)
Comment (initseis) [reply src/components/Calculator.js:202]: Hi, Arslan. Now I'm displaying the error when it happens.

**Comment by Davidosky007** [comment]
## Hi @sevinchek 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by initseis** [comment]
Thanks, @Davidosky007.

**Pull request merged**